site-src/guides/index.md

```diff
@@ -18,22 +18,26 @@ This quickstart guide is intended for engineers familiar with k8s and model serv
 
 ### Deploy Sample Model Server
 
-   Two options are supported for running the model server:
+   Three options are supported for running the model server:
 
-   1. GPU-based model server.  
+   1. GPU-based model server.
       Requirements: a Hugging Face access token that grants access to the model [meta-llama/Llama-3.1-8B-Instruct](https://huggingface.co/meta-llama/Llama-3.1-8B-Instruct).
 
-   1. CPU-based model server (not using GPUs).  
-      The sample uses the model [Qwen/Qwen2.5-1.5B-Instruct](https://huggingface.co/Qwen/Qwen2.5-1.5B-Instruct).  
+   1. CPU-based model server (not using GPUs).
+      The sample uses the model [Qwen/Qwen2.5-1.5B-Instruct](https://huggingface.co/Qwen/Qwen2.5-1.5B-Instruct).
 
-   Choose one of these options and follow the steps below. Please do not deploy both, as the deployments have the same name and will override each other.
+   1. [vLLM Simulator](https://github.com/llm-d/llm-d-inference-sim/tree/main) model server (not using GPUs).
+      The sample is configured to run the [meta-llama/Llama-3.1-8B-Instruct](https://huggingface.co/meta-llama/Llama-3.1-8B-Instruct) model.
```

Comment:
suggest using "simulate" rather than "run". the simulator doesn't run the model, it simulates it.
```suggestion
      The sample is configured to simulate the [meta-llama/Llama-3.1-8B-Instruct](https://huggingface.co/meta-llama/Llama-3.1-8B-Instruct) model.
```

Reply:
Commit [c5029cf](https://github.com/kubernetes-sigs/gateway-api-inference-extension/commit/c5029cf52e82a5e3dc3f1d317717afe386e57e4c) resolves this comment.